### PR TITLE
Openshift serverless prow

### PR DESCRIPTION
**Pull request by mgencur**
CI for the respective repository is being setup in https://github.com/openshift-knative/serverless-operator/pull/5

I'll update the owners when I have the proper list. ASAP

### CODE REVIEW

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -469,6 +469,7 @@ tide:
     - openshift/verification-tests
     - openshift/vertical-pod-autoscaler-operator
     - openshift/windows-machine-config-operator
+    - openshift-knative/serverless-operator
```
Comment (stevekuznetsov) [inline core-services/prow/02_config/_config.yaml]: This only adds `tide` for release branches -- you want another one up as well

Comment (mgencur) [reply core-services/prow/02_config/_config.yaml]: Really? This section has `excludedBranches` so I supposed all other branches would be included. Anyway, we currently only have the `master` branch. What should I do to enable the tide plugin for the master branch? Thanks

**Comment by mgencur** [comment]
The plugins are now configured for the whole org and OWNERS are present as well.

File: core-services/prow/02_config/_config.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply core-services/prow/02_config/_config.yaml]: Err so it only adds it for non-release, my bad. Do you not expect to have release branches?

Comment (mgencur) [reply core-services/prow/02_config/_config.yaml]: We expect to have release branches later. But they will be probably named differently from what we have in `excludedBranches`. So, I believe we should be fine. Please correct me if I'm missing something here.

**Comment by mgencur** [comment]
@stevekuznetsov thanks for the review. I don't think we're expecting to follow a bugzilla workflow here.

/hold cancel

**Pull request merged**